lib/erb_lint/cache.rb

```diff
@@ -9,6 +9,8 @@ def initialize(config, cache_dir = nil)
       @cache_dir = cache_dir || CACHE_DIRECTORY
       @hits = []
       @new_results = []
+      @gemfile_lock = File.read("Gemfile.lock") if File.exist?("Gemfile.lock")
```

Comment:
It looks like this will also bust the cache whenever any project dependencies change, even if the rubocop version did not. So, while I think this will work to achieve the desired effect not reusing the cache between different rubocop versions, it will also make running the erblint slower than it could/should be by not utilizing the cache when it could.

Reply:
Correct. I was aware and okay with that tradeoff, erring on the side of caution. 

Alternatively, you would have to hash the versions for `rubocop` and all `rubocop` related plugins in `Gemfile.lock`. Doable but I didn't want to spend any more time on it.

I find this simple and effective, with the caveat that non-rubocop updates will break the cache but if you want to write up the alternative solution that would be great.